data/transactions/logic/assembler.go

```diff
@@ -1727,12 +1727,13 @@ func addPseudoDocTags() {
 				continue
 			}
 			msg := fmt.Sprintf("`%s` can be called using `%s` with %s.", spec.Name, name, joinIntsOnOr("immediate", i))
-			desc, ok := opDocByName[spec.Name]
-			if ok {
-				opDocByName[spec.Name] = desc + "<br />" + msg
+			desc := opDescByName[spec.Name]
```

Comment:
I guess the `ok` here goes away for `opDescByName` takes any valid spec name, but previous `opDocByName` accepts a subset of all the valid spec names.

Reply:
The `ok` was checking to see if there was a description in `opDocByName`, that is now equivalent to having a non-blank `.Short` field in `opDescByName`.  So the comparison of `desc.Name` to "==" should be the same.

On the other hand, this code should probably yell and scream if there's no `.Short` description, it probably means the opcode isn't described at all, and that's likely a bug. Let me check it out.

Reply:
See latest commit.  Removed the code path that was there only for testing, but was never actually used.